.travis.yml

```diff
@@ -59,6 +59,15 @@ before_script:
     fi
 
 script:
+# !!!WARNING!!!
```

Comment:
Let's move this above `script:` so that we aren't splitting it from the list of definitions.